app/models/labware_creators/tagging/tag_collection.rb

```diff
@@ -89,7 +89,7 @@ def acceptable_templates
     # Returns true if the given template is in the approved list
     # or the approved list is empty. Returns false otherwise.
     #
-    # @param [Limber::TagLayoutTemplate] template The template to check
+    # @param [TagLayoutTemplates] template The template to check
```

Comment:
Plural type name for template param of the type Sequencescape::Api::V2::TagLayoutTemplate in yard.